airflow/utils/context.pyi

```diff
@@ -48,7 +48,7 @@ class VariableAccessor:
 class ConnectionAccessor:
     def get(self, key: str, default_conn: Any = None) -> Any: ...
 
-class Context(TypedDict, total=False):
+class Context(TypedDict):
```

Comment:
Not sure of this one cc: @uranusjr

Reply:
Why would you need that ? This means that we can omit any of the fields but IMHO that would be great if we can make assumption that all fields are present in Context  - so that whenever we need, we just add empty/default values for them whenever we find it necessary.

Reply:
This shouldn’t really matter because totality is checked on initialisation, and we don’t actually instantiate the context object directly, only force-cast a dict into one. And I chose `total=False` because it is arguably more correct semantically—the context does not always contain all the keys listed in this fake TypedDict.

What is the reason behind this change?

Reply:
This error:
```
airflow/utils/context.pyi:51: error: Unexpected keyword argument "total" for "__init_subclass__"
of "object"
    class Context(TypedDict, total=False):  
    ^
Found 1 error in 1 file (checked 60 source files)
```